src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -18,106 +18,119 @@
                     <div class="slds-section slds-is-open">
                         <h3 class="slds-section__title slds-theme_shade">
-                            <span class="slds-truncate slds-p-horizontal_small">{customLabels.scheduleSectionHeader}</span>
+                            <span class="slds-truncate slds-p-horizontal_small">{customLabels.statusSectionHeader}</span>
                         </h3>
                         <div aria-hidden="false" class="slds-section__content">
 
-                            <lightning-layout horizontal-align="spread" multiple-rows="true">
-                                <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
-                                    <lightning-input-field field-name={fields.amount.apiName} required>
+                            <lightning-layout horizontal-align="spread" multiple-rows="true" if:true={isFormReady}>
+                                <lightning-layout-item padding="horizontal-small" size="6" if:true={fields.status}>
+                                    <lightning-input-field field-name={fields.status.apiName} required>
                                     </lightning-input-field>
                                 </lightning-layout-item>
 
-                                <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
-                                    <lightning-input-field field-name={fields.paymentMethod.apiName}>
+                                <lightning-layout-item padding="horizontal-small" size="6" if:true={fields.statusreason}>
+                                    <lightning-input-field field-name={fields.statusreason.apiName}>
                                     </lightning-input-field>
                                 </lightning-layout-item>
+                            </lightning-layout>
 
-                                <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
-                                    <template if:true={isMultiCurrencyEnabled}>
+                        </div>
+                    </div>
+                </template>
+
+                <div class="slds-section slds-is-open">
+                    <h3 class="slds-section__title slds-theme_shade">
+                        <span class="slds-truncate slds-p-horizontal_small">{customLabels.scheduleSectionHeader}</span>
+                    </h3>
+                    <div aria-hidden="false" class="slds-section__content">
+
+                        <lightning-layout horizontal-align="spread" multiple-rows="true" if:true={isFormReady}>
+                            <lightning-layout-item padding="horizontal-small" size="6" small-device-size="6" if:true={fields.amount}>
+                                <lightning-input-field field-name={fields.amount.apiName} required>
+                                </lightning-input-field>
+                            </lightning-layout-item>
```

Comment:
The amount input field will be null out when there is a validation error on saving the record.

Reply:
Good catch. Not quite sure how to handle this one though.